book/src/bridge/extern_rustqt.md

```diff
@@ -131,7 +131,19 @@ Where `<Property>` is the name of the property.
 
 These setters and getters assure that the changed signal is emitted every time the property is edited.
 
-> Note that in the future it will be possible to specify custom getters and setters
+It is also possible to specify custom getters, setters and on-changed functions, using flags passed like so:
```

Comment:
```suggestion
It is also possible to specify custom getters, setters and notify signals, using flags passed like so:
```

It's not really custom on_ functions, but a custom signal that you can specify.

Reply:
Done